src/components/ChatEntry.js

```diff
@@ -1,22 +1,51 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+import { useState } from 'react';
+
+
+const ChatEntry = ({
+  id, 
+  sender, 
+  body, 
+  timeStamp, 
+  liked,
+  likesCountCallBack,
+}) => {
+
+  const [isliked, setIsLiked] = useState(liked)
```

Comment:
The state should not exist both on the child and the parent. We can just use the `liked` prop directly, there is no need to have another state variable here.